modules/engage-theodul-core/src/main/resources/ui/js/engage/models/meInfo.js

```diff
@@ -20,162 +20,78 @@
  */
 /*jslint browser: true, nomen: true*/
 /*global define, CustomEvent*/
-define(['jquery', 'backbone'], function($, Backbone) {
-    "use strict";
+define(['jquery', 'backbone', 'js-yaml.min'], function ($, Backbone, jsyaml) {
+  "use strict";
 
-    var prop_shortcut = "player.shortcut.",
-        prop_shortcut_sequence = "player.shortcut-sequence",
-        prop_allowedtags = "player.allowedtags",
-        prop_allowedformats = "player.allowedformats",
-        prop_mastervideotype = "player.mastervideotype",
-        prop_positioncontrols = "player.positioncontrols",
-        prop_layout = "player.layout",
-        prop_focusedflavor = "player.focusedflavor",
-        prop_logo_player = "logo_player",
-        prop_logo_mediamodule = "logo_mediamodule",
-        prop_link_mediamodule = "link_mediamodule",
-        prop_show_embed_link = "show_embed_links",
-        prop_matomo_server = "player.matomo.server",
-        prop_matomo_site_id = "player.matomo.site_id",
-        prop_matomo_heartbeat = "player.matomo.heartbeat",
-        prop_matomo_notification = "player.matomo.notification",
-        prop_matomo_track_events = "player.matomo.track_events",
-        prop_hide_video_context_menu = "player.hide_video_context_menu",
-        ready = false,
-        positioncontrols = "";
+  var configURL = "/ui/config/theodul/config.yml"
```

Comment:
There should be a comma here to not make the other vars global:

```suggestion
  var configURL = "/ui/config/theodul/config.yml",
```

Reply:
@lkiesow I found the issues, it was not only that but also after I activated ESlint with the profile provided in Opencast, there was missing also a semicolon ";" at the end of the file, also fixed.

I had to change in the YAML config file some integers into Characters. Now works as it should.

PS: Sorry for the late, I fixed my mac during the DACH meeting and today I had time for this.